### ISSUE TITLE
Chat Error

### ISSUE THREAD

**Issue by MothGirlMusic** [Zrips/CMI#689]
**Description of issue or feature request:**
when the player isn't on the same server as the one he's talking to on global chat from CMI, a weird issue of chat doubling happens.

#### DELETE IF YOU ARE SUBMITTING A FEATURE REQUEST ####

**ERROR (DELETE IF YOU HAVE NO ERROR):**
There is no error in the console when this happens, but here is a picture of the issue:
![screen shot 2018-08-01 at 12 20 06 pm](https://user-images.githubusercontent.com/1746550/43534409-45528290-9585-11e8-99ab-c7b79742de73.png)

**CONFIG SECTION (DELETE IF NOT RELEVANT):**
```
  # Enables DiscordSRV plugin support for messages
  DiscordSRVSupport: true
```
```
# Do you want to enable private messaging over bungeecord
  BungeeMessages: true
  # Do you want to enable public messaging over bungeecord
  BungeePublicMessages: true
  # Used for simple chat messages. Optional variables: {world} {prefix} {suffix} {group}. Supporting PlaceHolderAPI variables like %player_server%
  GeneralFormat: '[&b%player_server%][%towny_town%]{prefix}&f{displayName}&7: &r{message}'
  # Defines range of regular messages to travel
  # Set to -1 to disable range restriction
  GeneralRange: -1
  # Defines range of shout messages to travel
  # Shout messages should start with ! and player should have cmi.shout permission
  # GeneralRange should be enabled
  # set to 0 to shout across all worlds, -1 to disable
  ShoutRange: 200
  # Defines cost for each shout message
  ShoutCost: 0
  # Use numeric increments to separate groups from each other. If player has more than one, then line with higher number will be used
  # Add as many lines as you need too
```

**Cmi Version (using`/cmi version`):** 
8.0.4.3
**Server Type (Spigot/Paperspigot/etc):** 
Spigot servers in a Waterfall Bungeecord network.
**Server Version (using `/ver`):** 
git-Spigot-69774b3-6997012 (MC 1.13)

![screen shot 2018-08-01 at 12 21 29 pm](https://user-images.githubusercontent.com/1746550/43534560-ad23959e-9585-11e8-8060-5880f9785924.png)

as a side note, this hasn't happened before, its always said the version but now it looks like its showing me the variable thats supposed to show the version.

Proof of updating vault and CMI and the issue still persists:
![screen shot 2018-08-08 at 2 14 03 am](https://user-images.githubusercontent.com/1746550/43819456-f9d82de8-9ab0-11e8-9d6b-f1d14e6038c4.png)

**Comment by MothGirlMusic** [comment]
@Zrips this issue still persists with the newest version.

i've tried disabling DiscordSRV hooks in the CMI config to no avail. same issue. i've spoken to the DiscordSRV developers and they said its part of how your plugin handles chat.

**Comment by MothGirlMusic** [comment]
i belive this issue started in 8.0.3.0

**Comment by mercurialmusic** [comment]
Are you using another chat plugin? This could be a few different things.

Also you might try using his Vault build, which is more compatible: http://ltcraft.lt/wp-content/uploads/2018/04/Vault1.6.10.jar -- the one you're using is out of date.

Re: your review on Spigot, 7 days during a major update for such a large plugin and one of the busiest devs in the Spigot community is hardly long enough. He even says on the front page, if you're a 1.12 server having issues with the 8.0 builds to downgrade to 7.0 in the meantime while all the bugs get worked out. Bugs should be expected during this time.

**Comment by MothGirlMusic** [comment]
no i'm using CMI and DiscordSRV as my chat plugins. nothing else. though, i have tried other discord forwarder plugins. same thing happens and all the authors of those plugins say CMI is doing it.

i am currently using that version of vault, @mercurialmusic i updated the day after trying to fix stuff and thought i'd see if that fixes anything. it doesn't. also the one was using in the pic is the latest "actual" legit official version that was released via vault's bukkit and spigot pages. and not this special version you linked that i am using now. 

and lastly, my 1.13 server has the same issue and the issue only started on that update for 8.0.3.0. so being that i'm having the issue on my 1.13 server, its still relevant and downgrading to 7.0 would break my CMI install on my 1.13.

**Comment by Zrips** [comment]
I have noticed that you do have bungeemessages enabled and those messages looks like coming from another server, is this correct?

**Comment by MothGirlMusic** [comment]
woo! didn't have time to even answer before the author pushed a fix out in the update. thank you for seeing my issue and thanks for fixing it! i've redone my rating to 5 stars for this update! very happy with it and that solved all of my issues. thanks. i guess i'll close this now since its been fixed in 8.0.4.4
